### PR TITLE
fix: issue #4690: html element with overflow-y auto instead of scroll css

### PR DESCRIPTION
**Pull request by Miltonbhowmick**
### Description
This commit addresses the issue titled 'There is an scroll bar without having scrolling content https://github.com/monkeytypegame/monkeytype/issues/4690' by modifying the CSS of html element in `core.scss` file.
I used `overflow-y: auto` by changing `overflow-y: scroll`

Closes https://github.com/monkeytypegame/monkeytype/issues/4690

### CODE REVIEW

**Comment by Miodec** [comment]
As mentioned in the issue, the scrollbar always being there is intended.

File: frontend/.firebaserc_example [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
   "projects": {
-    "default": "your-firebase-project-id"
+    "default": "monkeytype-c4b6a"
   }
-}
+}
```
Comment (Miodec) [inline frontend/.firebaserc_example:5]: Also for future reference, you shouldnt commit this.

File: frontend/src/ts/constants/firebase-config-example.ts [hunk 2]
```diff
@@ -1,14 +0,0 @@
...
-
-export const firebaseConfig = {
-  apiKey: "",
-  authDomain: "",
-  databaseURL: "",
-  projectId: "",
-  storageBucket: "",
-  messagingSenderId: "",
-  appId: "",
-};
```
Comment (Miodec) [inline frontend/src/ts/constants/firebase-config-example.ts:14]: You shouldnt remove this file either.

**Comment by Miltonbhowmick** [comment]
> As mentioned in the issue, the scrollbar always being there is intended.

This shouldn't be there. If there are more content in the page scroll bar will be appeared. That's why i added `overflow-y: auto` instead of `overflow-y: scroll`

**Pull request closed without merging**